CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 ### Added
 - Document HTTP/2 support ([#330](https://github.com/opensearch-project/opensearch-java/pull/330))
 - Add support for phase_took & search_pipeline request params ([#1036](https://github.com/opensearch-project/opensearch-java/pull/1036))
+- Add an interface PlainJsonSerializable extending JsonpSerializable having default serialize method.([#1064](https://github.com/opensearch-project/opensearch-java/pull/1064))
```

Comment:
Remove the period and add a space to match other CHANGELOG lines.

I think we should explain the purpose of this `Add support for serializing requests and responses to plain JSON and nd-JSON`?

Reply:
Sorry for the delay, I have made a change on this, tried to keep it short and clear in meaning, please let me know if any work is remaining , and when can we merge this PR and [withJson ](https://github.com/opensearch-project/opensearch-java/pull/657/files), very much needed .
thanks a lot  :)